pandas/tests/test_nanops.py

```diff
@@ -817,6 +831,99 @@ def test_non_convertable_values(self):
                           lambda: nanops._ensure_numeric([]))
 
 
+class TestNanvarFixedValues(tm.TestCase):
+
+    def setUp(self):
+        # Samples from a normal distribution.
+        self.variance = variance = 3.0
+        self.samples = self.prng.normal(scale=variance ** 0.5, size=100000)
+
+    def test_nanvar_all_finite(self):
+        samples = self.samples
+        actual_variance = nanops.nanvar(samples)
+        np.testing.assert_almost_equal(
```

Comment:
I resorted to using the Numpy variants because I needed to set the decimal precision to something low; is there a Pandas version that also allows the precision to be adjusted?